src/beautifiers/terraformfmt.coffee

```diff
@@ -0,0 +1,39 @@
+###
+Requires terraform installed
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Terraformfmt extends Beautifier
+  name: "terraformfmt"
+  link: "https://www.terraform.io/docs/commands/fmt.html"
+  isPreInstalled: false
```

Comment:
`isPreInstalled` should be removed. See https://github.com/Glavin001/atom-beautify/blob/49982c6ce7b0c8f66042c2eddbc49d1441944023/src/beautifiers/beautifier.coffee#L40-L41